requirements.txt

```diff
@@ -1,14 +1,48 @@
+astroid==3.0.3
 certifi==2023.11.17
+charset-normalizer==3.3.2
 click==8.1.7
+dateparser==1.2.0
+DateTime==5.4
+decorator==5.1.1
+dill==0.3.7
 elastic-transport==8.11.0
 elasticsearch==8.11.1
 elasticsearch7==7.13.0
+expandvars==0.6.5
 fmatch==0.0.4
-numpy==1.26.3
-pandas==2.1.4
+gevent==23.9.1
+greenlet==3.0.3
+hunter @ git+https://github.com/datastax-labs/hunter.git@8ff166979d000780ad548e49f006ef2a15d54123
```

Comment:
what is this 8ff tag at the end? Should we specify a specific version/release thats more known instead

Reply:
would we be ok with just using the master branch?

Reply:
https://github.com/datastax-labs/hunter/issues/15 -- lets see if this can help us...